### ISSUE TITLE
Screenshots in Accessibility are taken before page loads

### ISSUE THREAD

**Issue by johnmccarrick** [cypress-io/cypress#30967]
As far as I'm aware we don't have much control over accessbility tests, but I have noticed that we will get flagged for a violation on an element that hasn't actually loaded yet and has the correct role. For example, our code was flagged with this critical violation: "Expecting ARIA child role to be added: tab", but when viewing the report the screenshot was captured before the page finished loading and when manually checking that element it does have the role: tab enabled

**Comment by marktnoonan** [comment]
Hi @johnmccarrick, thanks for reporting this! Please use the "Submit Feedback" button in Cypress Accessibility where you see this violation and that will give us enough information to look at your specific example.

False positives are rare, we have some documentation here about them: https://docs.cypress.io/accessibility/guides/accessibility-automation#False-positives - we investigate any time an issue is reported and there is usually a way to avoid these appearing in your reports.

**Comment by jennifer-shehane** [comment]
Closing this issue here as this repo is for issues and feature request of the Cypress App.

**Comment by johnmccarrick** [comment]
@marktnoonan Thanks Mark! I just submitted this as feedback
